weasyprint/formatting_structure/build.py

```diff
@@ -311,9 +319,11 @@ def marker_to_box(element, state, parent_style, style_for, get_image_from_uri,
         # See https://drafts.csswg.org/css-pseudo-4/#marker-pseudo
         marker_box.style['position'] = 'absolute'
         if parent_style['direction'] == 'ltr':
+            marker_box.style['left'] = 'auto'
```

Comment:
Why is this line needed?

Reply:
After applying this patch it seems the only way for `direction: ltr` to work is to set left to auto.